Calculators/Percentage-Calculator/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Enhanced Percentage Calculator</title>
+</head>
+<body>
+    <div>
+        <h1>~PC~</h1>
```

Comment:
Pls do remove this and add your calculator in main `index.html` file @itsmeabhishek03